### PR TITLE
Add mypy typing to test_mediatorstate.py

### PR DESCRIPTION
**Pull request by karlb**
I was annoyed by the lack of testing when I tried to modify a test.

### CODE REVIEW

File: raiden/tests/utils/factories.py [hunk 1]
```diff
@@ -351,9 +351,9 @@ def make_hop_to_channel(channel_state: NettingChannelState = EMPTY) -> HopState:
 # In this module constants are in the bottom because we need some of the
 # factories.
 # Prefixing with UNIT_ to differ from the default globals.
-UNIT_SETTLE_TIMEOUT = 50
-UNIT_REVEAL_TIMEOUT = 5
-UNIT_TRANSFER_AMOUNT = 50
+UNIT_SETTLE_TIMEOUT = BlockExpiration(50)
+UNIT_REVEAL_TIMEOUT = BlockExpiration(5)
```
Comment (palango) [inline raiden/tests/utils/factories.py:355]: Those should be `BlockTimeout`, right?

File: raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py [hunk 2]
```diff
@@ -103,102 +121,114 @@ def make_route_from_channelstate(channel_state):
...
 
 
 def test_is_channel_usable_for_mediation():
     """ Check rules that determine if a channel can be used for transfers """
-    reveal_timeout = 30
-    timeout_blocks = reveal_timeout + 10
-    amount = UNIT_TRANSFER_AMOUNT
+    reveal_timeout = BlockTimeout(30)
+    timeout_blocks = BlockTimeout(reveal_timeout + 10)
+    amount = TokenAmount(UNIT_TRANSFER_AMOUNT)
```
Comment (palango) [inline raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py:177]: This shouldn't be necessary

**Review by palango — APPROVED** [review]
🏅

File: raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py (hunk 2 above)
Comment (karlb) [reply raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py:177]: Yes, I got confused by a test that used this as an expiration value.

File: raiden/tests/utils/factories.py (hunk 1 above)
Comment (karlb) [reply raiden/tests/utils/factories.py:355]: Yes, I got confused by a test that used this as an expiration value.

**Pull request merged**